src/UpdatedPackagesCollector.js

```diff
@@ -28,9 +28,24 @@ export default class UpdatedPackagesCollector {
 
   collectUpdatedPackages() {
     logger.info("Checking for updated packages...");
-    progressBar.init(this.packages.length);
+
+    const currentBranch = GitUtilities.getCurrentBranchDescription();
+
+    if (currentBranch !== "master") {
```

Comment:
I don't think we should be prejudicing `master` here. It's also not Lerna's job to inform the user which branch they are on.